presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1138,14 +1137,6 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
         HiveOutputTableHandle handle = (HiveOutputTableHandle) tableHandle;
 
         List<PartitionUpdate> partitionUpdates = getPartitionUpdates(fragments);
-
-        if (isWritingStagingFilesEnabled(session)) {
```

Comment:
Send out a separate PR to remove these 😃

Reply:
We could put them in a separate commit, but maybe not separate PR. The following code would have to be removed and re-introduced.

```
        if (isWritingStagingFilesEnabled(session)) {
             writeFileName = ".tmp.presto." + filePrefix + "_" + randomUUID() + extension;
         }
```